requirements-dev.txt

```diff
@@ -1,14 +1,14 @@
 -r requirements.txt
-black
-Faker
-flake8
-ipython
-model_bakery
-pre-commit
-pytest
-pytest-cov
-pytest-django
-sphinx
+black==21.9b0
+Faker==8.13.2
+flake8==3.9.2
+ipython==7.27.0
```

Comment:
Looks like `ipython==7.27.0` is not supported for Python 3.6. That's why CI is failing for Python 3.6

Reply:
Sure, I've fixed this.